bootstrap/src/test/kotlin/jp/co/soramitsu/bootstrap/IrohaTest.kt

```diff
@@ -185,7 +186,23 @@ class IrohaTest {
                 respBody.contains(pubKey),
                 "pubKey not exists in block when should for account ${it.accountName}@${it.domainId} pubKey:${it.pubKeys[0]}"
             )
+            D3TestContext.d3neededAccounts.forEach {account ->
+                account.roles.forEach {role ->
+                    val testValue = "{\\\"appendRole\\\":{\\\"accountId\\\":\\\"" + account.id + "\\\",\\\"roleName\\\":\\\"" + role + "\\\"}}"
```

Comment:
I think `String.irohaEscape()` can add extra slashes in json for you.

Reply:
Thankyou